apis/oauth/src/main/java/org/jclouds/oauth/v2/domain/ClientCredentialsClaims.java

```diff
@@ -49,6 +50,7 @@
     public abstract long nbf();
 
     /** "JWT ID", a unique identifier for the JWT. */
+    @Nullable
```

Comment:
Should this also annotate the `create` method with `@Nullable` as the [AutoValue documentation suggests](https://github.com/google/auto/blob/master/value/userguide/howto.md#nullable)?

Reply:
Modified as per the comment and commit the changes.